addons/website_sale/models/res_config.py

```diff
@@ -122,3 +139,24 @@ def _onchange_sale_pricelist_setting(self):
                     'group_sale_pricelist': True,
                     'group_pricelist_item': True,
                 })
+
+    @api.depends('group_show_price_total')
+    def _compute_sale_show_tax(self):
+        self.sale_show_tax = 'total' if self.group_show_price_total else 'subtotal'
+
+    @api.onchange('sale_show_tax')
+    def _onchange_sale_tax(self):
```

Comment:
Same as above for coding guidelines note: we order by fields, _compute, _onchange, with same order as fields.